graphql-groups.gemspec

```diff
@@ -44,8 +44,8 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'rspec', '~> 3.0'
   spec.add_development_dependency 'rubocop', '~> 0.88'
   spec.add_development_dependency 'rubocop-rspec', '~> 1.42'
-  spec.add_development_dependency 'simplecov', '~> 0.18.5'
+  spec.add_development_dependency 'simplecov', '~> 0.21.2'
   spec.add_development_dependency 'sqlite3', '~> 1.4.2'
 
-  spec.add_dependency 'graphql', '~> 1', '> 1.9'
+  spec.add_dependency 'graphql', '~> 2.0.15', '> 1.9'
```

Comment:
There's no need to pin the version to 2.0.15 :) 

`spec.add_dependency` receives an array of conditions. I'm not sure why I pinned to version 1 back then. Since we want to use this with graphql two, you could just do this:  

```ruby
spec.add_dependency 'graphql', '> 1.9'
```
and that should do it :grin: